src/media/css/site.styl

```diff
@@ -257,6 +257,12 @@ a:active {
     }    
 }
 
+@media $wide-tablet {
+    .subheader.hidden {
```

Comment:
The hub-and-spoke header is only shown when `[data-page-type~="leaf"]`. You should be able to hide `[data-page-type~="leaf"] .subheader` by default and then show it on wide tablets.